packages/jest-console/package.json

```
@@ -17,9 +17,10 @@
     }
   },
   "dependencies": {
-    "@jest/source-map": "^25.2.6",
     "chalk": "^3.0.0",
-    "jest-util": "^25.3.0",
+    "jest-message-util": "^25.2.6",
+    "jest-types": "^25.2.6",
```

```suggestion
    "@jest/types": "^25.3.0",
```

and at the top.

`jest-util` is also the wrong version

---

Thanks for reminding about this! `dependencies` in `package.json` should always use package name while `references` in `tsconf.json` would be relative path. 

By the way, what's the reason some package names are scoped while others are not? Is it because packages published in the early days and not wanting to change package name?

---

We got the jest namespace pretty late, and to avoid confusion we haven't renamed any packages. So in general old packages are not scoped, but new ones are scoped